distribution/lib/Standard/Snowflake/0.0.0-dev/src/Internal/Snowflake_Dialect.enso

```diff
@@ -277,7 +281,7 @@ make_internal_generator_dialect =
     special_overrides = []
     other = [["RUNTIME_ERROR", make_runtime_error_op]]
     my_mappings = text + counts + stats + first_last_aggregators + arith_extensions + bool + date_ops + special_overrides + other
-    Base_Generator.base_dialect . extend_with my_mappings
+    Base_Generator.base_dialect_operations . extend_with my_mappings
```

Comment:
Feel like `Base_Genreator.dialect_operations` is good enough and we dont need to dupe base.

Reply:
It is from here. But inside Base_Generator it is less clear. Will think some more about Base_Generator. Still feel like it needs to be 2 files, just haven't found the dividing line yet